lib/irb/input-method.rb

```diff
@@ -299,6 +299,18 @@ def initialize
       if IRB.conf[:USE_AUTOCOMPLETE]
         Reline.add_dialog_proc(:show_doc, SHOW_DOC_DIALOG, Reline::DEFAULT_DIALOG_CONTEXT)
       end
+      if IRB.conf[:DIALOG_DEFAULT_BG_COLOR] && Reline.respond_to?('dialog_default_bg_color=')
```

Comment:
I think it's more idiomatic to use symbols rather than strings for `respond_to?`. E.g. `Reline.respond_to?(:dialog_default_bg_color=)`.